pkgs/misc/emulators/cdemu/vhba.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "vhba";
-  version = "20190831";
+  version = "20210418";
```

Comment:
Apparently the accompanying user space packages (also in `pkgs/misc/emulators/cdemu/`) have been updated as well: https://cdemu.sourceforge.io/project/#download

We may want to update these in lock-step, in order to avoid any compatibility issues.